apps/web-giddh/src/app/new-inventory/component/bulk-stock-edit/bulk-stock-edit.component.ts

```
@@ -192,23 +192,22 @@ export class BulkStockEditComponent implements OnInit, OnDestroy {
             select(select => select.inventory.bulkStock),
             takeUntil(this.destroyed$)
         ).subscribe((res: any) => {
-            if (res) {
-                this.isLoading = false;
+            this.isLoading = false;
+            if (res && res?.results) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)